scripts/setup_hooks.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPTS_DIR="scripts"
+HOOKS_DIR="scripts/hooks"
+
+current_directory=${PWD##*/}
+
+if [ $current_directory = $SCRIPTS_DIR ]
+then
+    cd ../
+fi
+
+hooks=($(ls scripts/hooks/ | sed 's/\.sh//'))
```

Comment:
Yeah but that results in 
```
scripts/hooks//pre-commit.sh
scripts/hooks//pre-push.sh
```

But with the `($(ls scripts/hooks/ | sed 's/\.sh//'))` it is just the file name. The `sed` is just removing the `.sh`